akka-stream-tests/src/test/scala/akka/stream/impl/GraphStageLogicSpec.scala

```diff
@@ -55,28 +54,35 @@ class GraphStageLogicSpec extends StreamSpec with GraphInterpreterSpecKit {
   object passThrough extends GraphStage[FlowShape[Int, Int]] {
     val in = Inlet[Int]("in")
     val out = Outlet[Int]("out")
+    in.id = 0
+    out.id = 1
```

Comment:
Yeap. It is re-enumerated in the create logics though, this was left over from the early trial and error.